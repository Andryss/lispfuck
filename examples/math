(printi (/
  (* 1 2 3 4 5)
  (* (mod 166 (- -5 -15)) (+ 2 -5 5 (- 10 10)))))
(printline)

(printi (= -9223372036854775808 (+ 1 9223372036854775807)))