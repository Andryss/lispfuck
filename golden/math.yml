in_source: |
  (printi (/
      (* 1 2 3 4 5)
      (* (mod 166 (- -5 -15)) (+ 2 -5 5 (- 10 10)))))
in_stdin: |

out_stdout: |
  10
